tests/hunspell_test.py

```diff
@@ -140,6 +140,14 @@ def test_stem(self):
         self.assertEqual(self.h.stem('dog'), ('dog',))
         self.assertEqual(self.h.stem('permanently'), ('permanent',))
 
+   def test_add(self):
```

Comment:
Missing a space here, which caused the tests to fail

Reply:
gotcha